### ISSUE TITLE
cannot be edited.

### ISSUE THREAD

**Issue by verydemo** [ONLYOFFICE/DocumentServer#1712]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
 Version 7.0.1  File cannot be edited.   Version 6.4.2 is editable.

**DocumentServer version:**
7.0.1

sample.docx
[sample.docx](https://github.com/ONLYOFFICE/DocumentServer/files/8409663/sample.docx)

**Comment by ShockwaveNN** [comment]
Hi, need some more details
What's the error?

Because I've opened this file on v7.0.1 without troubles (but it's opened quite slow, but seems it may use complex CJK fonts)

![image](https://user-images.githubusercontent.com/668524/161546980-e7f2d04c-55db-4db7-96da-5343682bf138.png)

**Comment by verydemo** [comment]
> Hi, need some more details What's the error?
> 
> Because I've opened this file on v7.0.1 without troubles (but it's opened quite slow, but seems it may use complex CJK fonts)
> 
> ![image](https://user-images.githubusercontent.com/668524/161546980-e7f2d04c-55db-4db7-96da-5343682bf138.png)

Files can be opened, but no content can be added. no error log

**Comment by ShockwaveNN** [comment]
Yeah, I can confirm that this file cannot be edited on v7.0.1 but I've rechecked it on v7.1.0 which currently in development and will be released in a month (I hope) and this file can be edited

So seems we already fixed this issue, you need to wait until v7.1 will be released to stable
